### PR TITLE
CDAP-3974: Add upgrade step for changes in Id serialization

### PR DESCRIPTION
**Pull request by jwang47**
- Upgrade stream state store for Id.Stream.toString changes

- [x] Tested in cluster

### CODE REVIEW

**Comment by jwang47** [comment]
https://issues.cask.co/browse/CDAP-3974
http://builds.cask.co/browse/CDAP-RBT539

File: cdap-master/src/main/java/co/cask/cdap/data/tools/StreamStateStoreUpgrader.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  @Inject
+  public StreamStateStoreUpgrader(LocationFactory locationFactory, HBaseTableUtil tableUtil,
+                                  NamespacedLocationFactory namespacedLocationFactory, Configuration conf) {
+    super(locationFactory, namespacedLocationFactory, tableUtil, conf);
+  }
+
+  @Override
+  protected TableId getTableId() {
+    return StreamUtils.getStateStoreTableId(Id.Namespace.DEFAULT);
```
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/StreamStateStoreUpgrader.java]: I think AbstractQueueUpgrader should now be refactored to upgrade all stream state store tables - not just the one in Default namespace.
Previously - for the upgrade tool in CDAP 2.8 - only the table in default namespace existed before upgrade, and so it only had to operate on that one table.

**Comment by anwar6953** [comment]
Code changes LGTM.
I think we should just verify that tables in multiple namespaces are upgraded.

**Comment by jwang47** [comment]
Verified that this works against multiple namespaces.

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**